resources/models/resource.py

```diff
@@ -262,6 +262,15 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                                       null=True, blank=True)
     reservable_min_days_in_advance = models.PositiveSmallIntegerField(verbose_name=_('Reservable min. days in advance'),
                                                                       null=True, blank=True)
+    cancellation_min_days_in_advance = models.PositiveIntegerField(
+        verbose_name=_('Reservation cancellation min. days in advance'),
+        null=True,
+        blank=True,
+    )
+    owner_can_cancel_reservation = models.BooleanField(
+        verbose_name=_('Owner can cancel manually confirmed reservation'),
```

Comment:
```suggestion
        verbose_name=_('Owner can manually cancel confirmed reservation'),
```
Would this be a better name?

Reply:
Ah, there is a reservation type which need manual confirmation. This line refers to such reservation.